graphviz.spec

```diff
@@ -366,11 +395,15 @@ Static development package for %{name}.
 %setup -q
 %apply_patches
 
-autoreconf -fiv
+#cat >./version.m4 <<EOF
+
+#autoreconf -fiv
```

Comment:
Probably better to just delete it than to comment it out, given it is replaced with autogen.sh later

Reply:
Deleted. I olny left it there just in case I needed to revert for some reason.